config/env.sh

```diff
@@ -13,9 +13,15 @@ MASTER2=172.16.0.23   # static IP of master2 VM
 WORKER0=172.16.0.24   # static IP of worker VM
 WORKER1=172.16.0.25   # static IP of worker VM
 WORKER2=172.16.0.26   # STATIC IP of worker VM
+WORKER3=172.16.0.27   # STATIC IP of worker VM
+WORKER4=172.16.0.28   # STATIC IP of worker VM
+WORKER5=172.16.0.29   # STATIC IP of worker VM
+STORAGE0=172.16.0.30   # STATIC IP of worker VM
+STORAGE1=172.16.0.31   # STATIC IP of worker VM
+STORAGE2=172.16.0.32   # STATIC IP of worker VM
 NETMASK=255.255.255.0 # /24 netmask for all above IP
 GATEWAY=172.16.0.1    # getway of network for all above IP
-PUBLICDNS=8.8.8.8     # DNS where PUBLIC names can be resolved
+PUBLICDNS=172.16.0.10     # DNS where PUBLIC names can be resolved
```

Comment:
i think the reason this worked for us was that we had dnsmasq setup as a resolver, and it would forward requests up to public DNS.   
i'm ok with your update to publicdns for now since it seems to be working

Reply:
All the OCP nodes were getting a DNS entry of 8.8.8.8 and therefore weren't going to the bastion for name resolution of cluster nodes. I can play a little more later but for now this works